### PR TITLE
Remove instance settings from sidebar

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

Addresses #8749. Removes "Instance status and settings" from sidebar and keeps settings in the site popover per design.

**With tooltip:**

## How did you test this code?
Not applicable.

### CODE REVIEW

**Review by clarkus — CHANGES_REQUESTED** [review]
This doesn't match the solution designed. The idea was to show org and project settings via the icons, but there was a secondary prominent link to the instance settings. 

Utility navigation
 
https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=7768%3A39486

**Comment by paolodamico** [comment]
@clarkus I was trying to follow the patterns of the other elements. The design is missing the other elements that we have in that section (instance version & license information), do we still want to include the other menu item there? Are you open for feedback on the design?

**Comment by clarkus** [comment]
> @clarkus I was trying to follow the patterns of the other elements. The design is missing the other elements that we have in that section (instance version & license information), do we still want to include the other menu item there? Are you open for feedback on the design?

It's the big blue instance settings link at the bottom. That other stuff is conditionally shown and the asset I linked isn't representative of all possible states. The problem that's been communicated to me is discoverability, which is why I'm suggesting this change for this specific issue.

**Comment by clarkus** [comment]
I updated the navigation diagrams to include all the states in case that helps unblock this change. The instance settings are more prominent here because this is the only place we link to them. The other settings are repeated in various places or can be accessed via multiple paths.

user   utility menu diagram

https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=7807%3A47508

**Comment by paolodamico** [comment]
> It's literally the big blue instance settings link at the bottom. That other stuff is conditionally shown and the asset I linked isn't representative of all possible states. The problem that's been communicated to me is discoverability, which is why I'm suggesting this change for this specific issue.

Chris, I've updated to match the blue link at the bottom. However I think the way you provided feedback above and in the PR could be more constructive and mindful of other perspectives. Not entirely sure if my points above were at all considered. Happy to chat sync if it helps.

**Comment by clarkus** [comment]
Thanks for updating. Apologies if my comments didn't feel constructive. I was trying to be clear, not aggressive. This is a great example of why looping in a designer for a review on UI changes can save us iterations in the future. Given a clear problem, we can speak factually and objectively about whether a problem is adequately solved or not. I appreciate you making these short term changes. Longer term we will clean this area up as it's somewhat becoming an instance management menu.

**Comment by clarkus** [comment]
Is the "instance status" link adjacent to the system status necessary? It's technically an accessibility concern when we link to the same asset multiple times using inconsistent labels. Could that link adjacent to the system status actually point to http://status.posthog.com/? If not, can we remove it and only show the big bold discoverable "instance settings" link at the bottom? I just don't think we need both links in this context.

**Review by tiina303 — APPROVED** [review]
LGTM, commented about the design in Figma

File: frontend/src/layout/navigation/TopBar/SitePopover.tsx
Comment (Twixes) [inline frontend/src/layout/navigation/TopBar/SitePopover.tsx:62]: This should be a `LemonButton` for better interactivity. It also supports the `to` prop, so there's no need for wrapping it with a link. The icon should be in the `icon` prop.

File: frontend/src/layout/navigation/TopBar/SitePopover.tsx
Comment (Twixes) [inline frontend/src/layout/navigation/TopBar/SitePopover.tsx:87]: Should be a button like above

File: frontend/src/layout/navigation/TopBar/SitePopover.tsx
Comment (Twixes) [inline frontend/src/layout/navigation/TopBar/SitePopover.tsx:61]: This tooltip should be placed on the left:
Screen Shot 2022-03-02 at 15 55 04

File: frontend/src/layout/navigation/TopBar/SitePopover.tsx
Comment (Twixes) [inline frontend/src/layout/navigation/TopBar/SitePopover.tsx:86]: Tooltip should be to the left

File: frontend/src/layout/navigation/TopBar/SitePopover.tsx
Comment (Twixes) [inline frontend/src/layout/navigation/TopBar/SitePopover.tsx:257]: This should be a `LemonButton` just like "Invite members" above.

**Review by Twixes — CHANGES_REQUESTED** [review]
All for this, but this introduces some UI inconsistencies in `SitePopover` (like "Invite members" and "Instance settings" behaving differently)

**Review by Twixes — APPROVED** [review]
Fixed the inconsistencies in implementation to move this forward faster. The design is essentially the same though. As for remaining UX concerns, I'm also not sure about having both "Instance status" and "Instance settings", but don't know what's the best solution (for "Instance status" we _can_ link to https://status.posthog.com on Cloud, but not on self-hosted).

**Comment by tiina303** [comment]
Could we also update the image in https://posthog.com/blog/the-posthog-array-1-33-0 so users know how to find it.

**Comment by paolodamico** [comment]
Thanks @clarkus for understanding and the feedback too. We all want to create the best product so really appreciate you pushing forward the quality. Thanks @Twixes too for the consistency fixes. For everyone else's context this is how the final version looks (when hovering over the instance settings menu item)

Screen Shot 2022-03-02 at 12 48 38 PM


Before merging this do we want to use the space for status to take to https://status.posthog.com on Cloud and remove the label on self-hosted? Or keep the label as is?

Sure thing @tiina303 I'll update once we finalize what to do with the instance status label.

**Comment by clarkus** [comment]
> Before merging this do we want to use the space for status to take to https://status.posthog.com/ on Cloud and remove the label on self-hosted? Or keep the label as is?

The label seems applicable regardless of how it's hosted, so maybe just show the status link for cloud users?

**Comment by paolodamico** [comment]
I really like the idea of having the status for PostHog Cloud in there but I just realized to properly show the "All systems operational" or something like "Downgraded experience" we would have to integrate with status.posthog.com to fetch any operational issues, so I would suggest keeping this for a subsequent improvement. Thoughts? Also for context, we would need to add the section to Cloud (this is how a normal Cloud user sees the popover now)




So for self-hosted for now we'll keep the label as it is today, right?

**Comment by clarkus** [comment]
I could see us swapping to use consistent icons for the management/settings links, but I don't think it's critical at this stage especially if this won't be released for a while. Probably fine for a follow up iteration.

**Review by clarkus — CHANGES_REQUESTED** [review]
Let's make the management / settings links for the the currently logged in users, and the current organization consistent with gear icons and tooltips. Other than that, my main concern is limiting to a single instance settings link and dropping the link adjacent to "all systems operational". 

Here's a quick redraw of a cloud utility menu and a hosted utility menu if it helps clarify intent. https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=7768%3A39486

Screen Shot 2022-03-02 at 11 11 10 AM

**Comment by tiina303** [comment]
why are we dropping the link from all systems operational - I want to be able to click and get to the place, where we show which systems are operational and which aren't. It should take me to the `/instance/status` page system overview tab.

**Comment by clarkus** [comment]
> why are we dropping the link from all systems operational - I want to be able to click and get to the place, where we show which systems are operational and which aren't. It should take me to the `/instance/status` page system overview tab.

Because the target of the Instance settings link goes to the same URL. It's an accessibility concern if we link to the same place in multiple ways using varying text labels. I am optimizing for linking to that location via one prominent link so we can improve discoverability.

**Comment by tiina303** [comment]
> Because the target of the Instance settings link goes to the same URL. It's an accessibility concern if we link to the same place in multiple ways using varying text labels. I am optimizing for linking to that location via one prominent link so we can improve discoverability.

It should be different - instance settings should go to `/instance/status` but settings tab

**Comment by tiina303** [comment]
furthermore currently if I see there's a problem and there isn't a link next to it I wouldn't know to click on the link below

Screen Shot 2022-03-02 at 20 53 56

^ here if we remove the link from next to the red error it will be very confusing to the users imho

**Comment by clarkus** [comment]
> It should be different - instance settings should go to /instance/status but settings tab

In my usage of the feature this is not how it works. It always goes to the default tab. Could it be a bug or some other issue? Even then, I'm not sure the targets are distinguishable enough to justify multiple links. 

> furthermore currently if I see there's a problem and there isn't a link next to it I wouldn't know to click on the link below
> ^ here if we remove the link from next to the red error it will be very confusing to the users imho

Thanks for the feedback, but I disagree. The need for going to this link isn't just in situations where there are errors. Those errors are conditionally show, so having a consistent place where we link to the page would be a better experience. For now, this feels like a clear enough improvement that's discoverable and placed in the page hierarchy accurately. Can we ship this iteration that improves discoverability and hierarchy concerns, and then queue up an iteration where we really try to nail instance management? I'm sure there are more problems to solve in here, so if we can track and describe those, we'll be in a better position to iterate and make more improvements.

**Comment by paolodamico** [comment]
Ok based on the conversation above I believe the conclusion was let's merge this in as it's already an improvement and continue the conversation on how to improve in the issue that @clarkus opened, so merging and we can keep working on the other issue.

**Pull request merged**
